src/convert_float_ibm_to_ieee32.c

```diff
@@ -1,4 +1,31 @@
-/***************************************************************************** 
+/*
+# Copyright (c) 2012 - 2015, The Met Office, UK
```

Comment:
1. Is this range of years really correct?
2. I'm not sure it makes sense to stick a Met Office copyright on a file which clearly isn't..!?